### ISSUE TITLE
Setting discoveryUrl is ignored

### ISSUE THREAD

**Issue by jakubkoje** [MarioArnt/azure-ad-jwt-lite#4]
When I set the discoveryUrl to v2 url, the URL is ignored and the library is still using the default URL.

```
const decoded = await verifyAzureToken(test, {
        discoveryUrl: 'https://login.microsoftonline.com/common/discovery/v2.0/keys',
      })
```

@MarioArnt it seems that the version in npm is not the same as here in repository.

**Comment by MarioArnt** [comment]
Hi @jakubkoje

Thank you for your feedback.

You're absolutely right, the version published was not built from v1.1.0 sources but from a previous version.

I'll republish it as v1.1.1 ASAP and set-up an automated publish process to avoid human (and thus error-prone) actions.

**Comment by MarioArnt** [comment]
Hello, 

The linked PR has setup semantic release on this repository and the issue is now fixed in >=1.1.1

Thanks again for this report :)

Mario
